main.cpp

```diff
@@ -59,10 +65,35 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
+    
+    int age = 25;
+    int year = 2021;
+    int numStereoChannels = 2;
+
+    double taxRate = 12.5;
+    double averageTemperature = 55.7;
+    double gpa = 3.8;
+
+    float width = 10.f;
+    float length = 30.f;
+    float peakQuality = 23.2f;
+    
+    char yes = 'y';
+    char no = 'n';
+    char assignOperator = '=';
+
+    bool isPlaying = false;
+    bool bufferFull = true;
+    bool hasInstance = true;
+
+    unsigned vectorSize = 6;
+    unsigned ipAddressFirstBit = 0;
+    unsigned ramSize = 8;
 
     
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+
+    ignoreUnused(age, year, numStereoChannels, taxRate, averageTemperature, gpa, width, length, peakQuality, yes, no, assignOperator, isPlaying, bufferFull, hasInstance, vectorSize, ipAddressFirstBit, ramSize);

```

Comment:
```
    b) at the end of the function, call ignoreUnused once and pass all of your variables to it. see line 71 for an example
```
You are calling `ignoreUnused` twice.  the instructions said to call it once.
- [ ] revise

Reply:
Whoops, silly rushing.